crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -1,3 +1,63 @@
+type ActiveJwk {
+	"""
+	The string (Issuing Authority) that identifies the OIDC provider.
+	"""
+	iss: String!
```

Comment:
The short field names here are designed to match the JWK (JSON Web Key) spec (RFC 7517), with the doc comments used to explain what they are.